From 3e63810210760998f197437e6c3fb82df87483b8 Mon Sep 17 00:00:00 2001
From: Dusty Mabe <dusty@dustymabe.com>
Date: Mon, 15 Jan 2024 16:14:50 -0500
Subject: [PATCH 4/4] fscache: add eviction log statement

This will help us know when the cache LRU policy is working.
---
 osbuild/util/fscache.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/osbuild/util/fscache.py b/osbuild/util/fscache.py
index 59039522..3aa6a1a4 100644
--- a/osbuild/util/fscache.py
+++ b/osbuild/util/fscache.py
@@ -1150,6 +1150,7 @@ class FsCache(contextlib.AbstractContextManager, os.PathLike):
                     # and update the cache size based on the size
                     # recorded inside "object.removing".
                     size = self._calculate_space(self._path(rpath))
+                    print(f"CACHE: evicting {rpath}")
                     self._rm_r_object(rpath)
                     self._update_cache_size(-size)
                     freed_so_far += size
-- 
2.43.0

